documentation/docs/contributing/development-guide/publishing.md

```diff
@@ -0,0 +1,54 @@
+---
+sidebar_position: 5
+---
+
+# Publishing
+
+## Github actions
+
+We use [GitHub Actions](https://docs.github.com/en/actions), as the [CI/CD pipeline](https://github.com/equinor/boilerplate-react-fastapi/actions).
+
+GitHub Actions makes it easy to automate all our software workflows. It build, test, and deploy the code right from GitHub.
+
+All the heavy lifting is done by GitHub. GitHub will simply use the files defined in `.github/workflows` to do the setup.
+
+We trigger the reusable workflow like this:
+
+![Github Actions](/img/github-actions.svg)
+
+
+Since we are using trunk based development, all pushes to main trigger a release to a test environment.
+
+Release to production is triggered when we merge in the auto-generated pull requests (that contains changelogs) that are created by the `release-please.yml` Github action. 
+
+### Reusable workflows
+
+A reusable workflow is just like any GitHub Actions workflow with one key difference - it includes a `workflow_call` trigger.
+
+* The `release.yaml` workflow will automatically create a pull request with an auto-generated changelog everytime code is pushed to the main branch. It will also at the same time bump version with standard-version and create a tagged release that can be used to deploy to production.
```

Comment:
```suggestion
* The `release-please.yaml` workflow will automatically create a pull request with an auto-generated changelog everytime code is pushed to the main branch. It will also at the same time bump version with standard-version and create a tagged release that can be used to deploy to production.
```